XpringKit/DefaultXpringClient.swift

```diff
@@ -178,4 +179,28 @@ extension DefaultXpringClient: XpringClientDecorator {
 
     return RawTransactionStatus(getTxResponse: getTxResponse)
   }
+
+  /// Check if an address exists on the XRP Ledger
+  ///
+  /// - Parameter address: The address to check the existence of.
+  /// - Throws: An error if there was a problem communicating with the XRP Ledger.
+  /// - Returns: A boolean if the account is on the blockchain.
+  public func accountExists(for address: Address) throws -> Bool {
+    guard
+      let _ = Utils.decode(xAddress: address)
+    else {
+      throw XRPLedgerError.invalidInputs("Please use the X-Address format. See: https://xrpaddress.info/.")
+    }
+    do {
```

Comment:
nit: I'd break this up with some vertical spacing. Generally, usually put an empty line after the closing bracket of guard clauses.

Reply:
done